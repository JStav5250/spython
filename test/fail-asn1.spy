str s = "hello"
char x = "c"

if false:
    s = 1.0

s[0] = x
