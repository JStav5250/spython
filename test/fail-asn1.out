TypeError: expression of type float cannot be assigned to variable 's' with explicit type str!!
